lib/repository/readonly-repository.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { of } from 'rxjs';
+import { Animal, Person, Thing } from '../mock/thing.entity';
+import { DataStoreService } from '../services/data-store.service';
+import { Paris } from '../services/paris';
+
+describe('ReadonlyRepository', () => {
+	describe('getModalData', () => {
```

Comment:
Why 'getModalData'? These tests test the modelWith configuration, maybe call the test suite that?